libraries/protocol/scorum_operations.cpp

```diff
@@ -336,5 +336,18 @@ void delegate_vesting_shares_operation::validate() const
     FC_ASSERT(is_asset_type(vesting_shares, VESTS_SYMBOL), "Delegation must be VESTS");
     FC_ASSERT(vesting_shares >= asset(0, VESTS_SYMBOL), "Delegation cannot be negative");
 }
+
+void create_budget_operation::validate() const
+{
+    validate_account_name(owner);
+    validate_permlink(content_permlink);
+    FC_ASSERT(is_asset_type(balance, SCORUM_SYMBOL), "Balance must be SCORUM");
```

Comment:
Not sure, but I thins message it should be SCR not SCORUM

Reply:
Why are we use SCORUM_SYMBOL, VESTS_SYMBOL?
Let 's change to SCR_SYMBOL, SP_SYMBOL

Reply:
And:

#define SCORUM_SYMBOL  (uint64_t(3) | (uint64_t('T') << 8) | (uint64_t('E') << 16) | (uint64_t('S') << 24) | (uint64_t('T') << 32) | (uint64_t('S') << 40)) ///< SCORUM with 3 digits of precision

Why TESTS?

Reply:
There are a lot of that in old code: 

FC_ASSERT(delegator != delegatee, "You cannot delegate VESTS to yourself");
    FC_ASSERT(is_asset_type(reward_scorum, SCORUM_SYMBOL), "Reward Scorum must be SCORUM");
    FC_ASSERT(is_asset_type(reward_vests, VESTS_SYMBOL), "Reward Scorum must be VESTS");
...

Reply:
for production SCORUM_SYMBOL -> "SCR"
for test net      SCORUM_SYMBOL -> "TESTS"

Reply:
Variable renaming is not an object of topic.